rero_mef/filter.py

```
@@ -42,3 +42,20 @@ def inner(values):
 
     return inner
+
+
+def multi_exists_filter(fields):
+    """Create a filter that checks if any of multiple fields exist.
+
+    :param fields: List of field names.
+    :returns: Function that returns a bool query with should clauses.
+    """
+
+    def inner(values):
```

Ruff (0.14.14)

Unused function argument: `values`

(ARG001)